src/encoded/static/components/antibody.js

```diff
@@ -67,18 +67,28 @@ const LotComponent = (props, reactContext) => {
     const geneTips = [];
     targetKeys.forEach((key, i) => {
         const scientificName = targets[key].organism.scientific_name;
-        const geneName = targets[key].gene_name;
+        const geneName = targets[key].genes.map(
+            gene => gene.symbol
+        );
 
         // Add to the information on organisms from the targets
         organismComponents.push(<span key={key}>{i > 0 ? <span> + <i>{scientificName}</i></span> : <i>{scientificName}</i>}</span>);
         organismTerms.push(`targets.organism.scientific_name=${scientificName}`);
         organismTips.push(scientificName);
 
         // Add to the information on gene names from the targets
-        if (geneName && geneName !== 'unknown') {
-            geneComponents.push(<span key={key}>{i > 0 ? <span> + {geneName}</span> : <span>{geneName}</span>}</span>);
-            geneTerms.push(`targets.gene_name=${geneName}`);
-            geneTips.push(geneName);
+        if (geneName.length) {
+            geneName.map(
+                symbol => geneComponents.push(
+                    <span key={key}>{geneComponents.length > 0 ? <span> + {symbol}</span> : <span>{symbol}</span>}</span>
+                )
+            );
+            geneTerms.push(
+                ...geneName.map(
+                    symbol => `targets.genes.symbol=${symbol}`
+                )
+            );
+            geneTips.push(...geneName);
```

Comment:
You’re cloning the geneName array into geneTips? You’ll probably want to comment this as this is a highly unusual way to do that, or you can change to:

geneTips = geneName.slice(0); // Very common
geneTips = [...geneName]; // Much less common, but MDN-approved